go/teams/create.go

```diff
@@ -10,16 +10,86 @@ import (
 	jsonw "github.com/keybase/go-jsonw"
 )
 
-func CreateRootTeam(ctx context.Context, g *libkb.GlobalContext, name string) (err error) {
-	defer g.CTrace(ctx, "CreateRootTeam", func() error { return err })()
+func CreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, impTeam keybase1.ImplicitTeamName) (res keybase1.TeamID, err error) {
+	defer g.CTrace(ctx, "CreateImplicitTeam", func() error { return err })()
 
-	g.Log.CDebugf(ctx, "CreateRootTeam load me")
-	me, err := libkb.LoadMe(libkb.NewLoadUserArg(g))
+	name, err := NewImplicitTeamName()
 	if err != nil {
-		return err
+		return res, err
 	}
+	teamID := RootTeamIDFromNameString(name)
+
+	// Load all the Keybase users
+	me, err := libkb.LoadMe(libkb.NewLoadUserArg(g))
+	if err != nil {
+		return res, err
+	}
+	var kbusers []*keybase1.UserPlusKeysV2AllIncarnations
+	for _, kbu := range impTeam.KeybaseUsers {
+		uid, err := g.GetUPAKLoader().LookupUID(ctx, libkb.NewNormalizedUsername(kbu))
```

Comment:
this is a slight bummer since you wind up loading the upak2 twice.  Can you make a ticket for `LoadV2` allows lookup by username? it's not the end of the world